src/components/DownloadLoginFile.js

```diff
@@ -68,7 +68,7 @@ class DownloadLoginFile extends Nimiq.Observable {
                 <span data-i18n="download-loginfile-download">Download Login File</span>
             </button>
             <span class="nq-label tap-and-hold" data-i18n="download-loginfile-tap-and-hold">Tap and hold image to download</span>
-            <button class="nq-button light-blue continue" data-i18n="download-loginfile-continue">Continue</button>
+            <button class="nq-button light-blue continue" data-i18n="download-loginfile-continue">Download</button>
```

Comment:
This is not correct. This button is there to continue to the next step, after the long-tap download of the image file is expected to be completed. This button is a fallback for iOS devices, which do not support the `download` attribute on links and need a manual button to _continue_.

Thus this button does _not_ download the image.

Reply:
Furthermore, the title for the Login File download pages is changed from `Download Login File` to `Save Account`, so this button should not say `Download` anymore.

Reply:
Again, @Tammoovernice, _this_ is _not_ the download button. The download button is above in line 68.

This button here is a button that appears only on iOS devices, where image _cannot_ be downloaded with a tap (because iOS does not support that), but _has to be_ downloaded with a long-tap. Because of all the things that can go wrong or the user aborting the save-file-dialog, we do not automatically continue to the next screen like we do for a regular click/tap on all other devices. This button is there to manually continue after downloading the image.